src/components/PageResultsNew/BuildMetadata.tsx

```diff
@@ -0,0 +1,181 @@
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import { useState } from 'react';
+import {
+    Badge,
+    CardBody,
+    DescriptionList,
+    DescriptionListDescription,
+    DescriptionListGroup,
+    DescriptionListTerm,
+    Tab,
+    Tabs,
+    TabTitleText,
+} from '@patternfly/react-core';
+import classNames from 'classnames';
+
+import styles from '../../custom.module.css';
+
+import { ExternalLink } from '../ExternalLink';
+
+interface BuildInfoProps {
+    buildId: number;
+    buildTime: string;
+    commit: string;
+    commitTime: string;
+    committerEmail?: string;
+    committerName?: string;
+    owner: string;
+}
+
+function BuildInfo(props: BuildInfoProps) {
+    const items = [
+        {
+            label: 'Build ID',
+            value: <ExternalLink href="#">{props.buildId}</ExternalLink>,
+        },
+        {
+            label: 'Git commit',
+            value: (
+                <ExternalLink
+                    className={styles['buildInfoCommitHash']}
+                    href="#"
+                >
+                    {props.commit}
+                </ExternalLink>
+            ),
+        },
+        {
+            label: 'Build owner',
+            value: <ExternalLink href="#">{props.owner}</ExternalLink>,
+        },
+        {
+            label: 'Committer',
+            value: (
+                <>
+                    {props.committerName || 'n/a'}
+                    &nbsp;&lt;
+                    {props.committerEmail || 'n/a'}
+                    &gt;
+                </>
+            ),
+        },
+        {
+            label: 'Build completed',
+            value: props.buildTime,
+            className: styles['buildInfoTimestamp'],
+        },
+        {
+            label: 'Commit time',
+            value: props.commitTime,
+            className: styles['buildInfoTimestamp'],
+        },
+    ];
+
+    return (
+        <DescriptionList
+            className={classNames(
+                'pf-u-px-xl',
+                'pf-u-py-lg',
+                styles['buildInfo'],
+            )}
+            columnModifier={{ default: '1Col', lg: '2Col' }}
+            isAutoColumnWidths
+            isCompact
+            isHorizontal
+        >
+            {items.map(({ className, label, value }, index) => (
+                <DescriptionListGroup key={index}>
+                    <DescriptionListTerm>{label}</DescriptionListTerm>
+                    <DescriptionListDescription className={className}>
+                        {value}
+                    </DescriptionListDescription>
+                </DescriptionListGroup>
+            ))}
+        </DescriptionList>
+    );
+}
+
+export function BuildMetadata(_props: {}) {
+    const [activeTabKey, setActiveTabKey] = useState('summary');
+
+    const buildProps = {
```

Comment:
Maybe mark mock-data with some comment to make sure to take care about it.

Reply:
But... looked above, it seems there are a lot of such places. 
So, ok to be without comment.

Reply:
I've added a couple more comments in the relevant places.